docs/tutorials/quickstart.md

```diff
@@ -24,4 +24,36 @@ By default, when you install OTIO you will only get the "Core" adapters, which i
 
 Once you have pip installed OpenTimelineIO, you should be able to run:
 
-    otioview path/to/your/file.edl
\ No newline at end of file
+    otioview path/to/your/file.edl
+
+
+# Developer Quickstart
+
+0.  Get the source and submodules:
+    + `git clone --recursive git@github.com:PixarAnimationStudios/OpenTimelineIO.git`
+
+1. To build OTIO only for C++ development (i.e. produce the OTIO C++ libraries and header files), then use cmake:
+    + `mkdir build`
+    + `cd build`
+    + `cmake .. {options}`
+    + `make install`
+    For {options} configure the variables PYTHON_EXECUTABLE, PYTHON_LIBRARY, and CMAKE_INSTALL_PREFIX appropriately.
+
+2. To build OTIO only for Python development, run the following command:
+   + `pip install .`
```

Comment:
Do you need to run this first? I think python setup.py install would do everything.

Reply:
Ah yes, that's right.